ci/prepareElmRelease.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const process = require('process');
+
+/**
+ A semantic release "prepare" plugin to update elm-package.json
+ */
+async function prepareElmRelease(config, context) {
+  const newVersion = context.nextRelease.version;
+  context.logger.log(
+    `Updating elm-package.json version number to ${newVersion}`
+  );
+  const elmPackageJson = JSON.parse(fs.readFileSync('elm-package.json'));
+  elmPackageJson.version = newVersion;
+  fs.writeFileSync('elm-package.json', JSON.stringify(elmPackageJson, null, 2));
```

Comment:
The actual committing of this change is handed by the Semantic Release framework?